fitbenchmarking/controllers/matlab_mixin.py

```diff
@@ -26,63 +30,78 @@ class MatlabMixin:
     Mixin class for matlab fitting software controllers
     """
 
+    incompatible_problems = ['mantid']
```

Comment:
since this is now set in `matlab_mixin`, do we still need to set this in the individual controller files?

Reply:
I had to do some git hunting to see why it's in individuals instead of the mixin, but it came up in a previous review and I think Rob thought it was clearer in individual controllers (although neither of us had particularly strong opinions).

You are right though that either place works so happy to remove this or clean out the other controllers.

Reply:
Discussion here: https://github.com/fitbenchmarking/fitbenchmarking/pull/979#discussion_r752378067

Reply:
I've removed the line now